core/src/main/java/novoda/lib/sqliteprovider/provider/SQLiteContentProviderImpl.java

```diff
@@ -63,10 +63,30 @@ protected SQLiteOpenHelper getDatabaseHelper(Context context) {
 
     @Override
     protected Uri insertInTransaction(Uri uri, ContentValues values) {
+        return insertInTransaction(uri, values, true);
```

Comment:
Are you sure about adding a flag here to switch behaviour? Wouldn't it be better to have something like:

```java
     @Override
     protected Uri insertInTransaction(Uri uri, ContentValues values) {
        Uri notificationUri =  insertSilently(uri, values, true);
        notifyUriChange(notificationUri);
        return notificationUri;
     }

```